src/elements/forms/element_dropdown.erl

```diff
@@ -135,7 +135,7 @@ is_selected(DropDownValue, #option{value=OptValue}) ->
     %% Finally, if none of the above short-circuits trip, then we can convert
     %% #option.value to binary and compare directly. If they match, then it's
     %% selected.
-    wf:to_binary(OptValue) =:= DropDownValue.
+    wf:to_unicode_binary(OptValue) =:= DropDownValue.
```

Comment:
While I don't think it's worth bringing up in this PR, I do like to take opportunities like this to note that in general, raw string equivalence isn't necessarily what we want. We usually want Unicode equivalence. In the dropdown
this isn't necessarily a risk because we aren't dealing with user entry, but if
we were we would probably want to use `string:equals(OptValue, DropDownValue, false, nfc)` so that varying representations of the same canonical characters compare equally.

The common trip-up is a case like the `ü` which either comes as two code points, `ü` (`u` + `̈`) U+75 U+0308, or as a single code point, `ü` U+FC, and both represent the same character but may appear in one context or another based on the OS and software settings.

The HTML I wrote [in this gist](https://gist.github.com/dmsnell/74696ab4e7e1dc07e15891f764684473) gives you an interactive way to explore these equivalence forms. Doubtful we would want "compatible" normalization here, but `NFC` normalization would generally be appropriate.

Reply:
I agree. That's a great suggestion!